topics/no-code/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: no-code-development-platform
```

Comment:
Let's drop this alias, it doesn't look like [any repos are using it yet](https://github.com/topics/no-code-development-platform).

```suggestion
```

We generally try to keep topics and aliases descriptive rather than prescriptive 😄

Reply:
Thank you @smashwilson — I've actually updated this to use `nocode` with no hyphen, as that seems to have quite a large number (100+) of repositories tagged as well.